internal/operatorcli/operator_cli.go

```diff
@@ -48,8 +48,16 @@ type avsClient struct {
 
 func RunEncrypt(c *cli.Context) error {
 	passphrase := c.String(flag.PassphraseFlag.Name)
+	blsPassphrase := c.String(flag.BlsPassphraseFlag.Name)
+	ecdsaPassphrase := c.String(flag.EcdsaPassphraseFlag.Name)
+
 	if passphrase == "" {
-		return cli.Exit("passphrase is required", 1)
+		if blsPassphrase == "" || ecdsaPassphrase == "" {
+			return cli.Exit("either passphrase or bls/ecdsa passphrase is required", 1)
+		}
+	} else {
+		blsPassphrase = passphrase
+		ecdsaPassphrase = passphrase
 	}
```

Comment:
need to change the order of checking the flags.
first checks the new flags (the configuration we want to have)
second check the backward compatbility